platforms/android/library/src/main/java/io/element/android/wysiwyg/utils/HtmlToSpansParser.kt

```diff
@@ -299,13 +299,14 @@ internal class HtmlToSpansParser(
         }
     }
 
+    private fun isLineBreak(node: Node?): Boolean {
+        return node is Element && node.tagName() == "br"
+    }
```

Comment:
I would have gone for a `private fun Element.isLineBreak(): Boolean` extension function, but this works too.